test/index.test.js

```diff
@@ -110,30 +110,39 @@ describe('mongodb-connection-model', function() {
     });
 
     describe('ATLAS - mongodb.net', function() {
-      var atlasConnection = 'mongodb://ADMINUSER:PASSWORD@' +
+      var atlasConnection = 'mongodb://ADMINUSER:<PASSWORD>@' +
           'a-compass-atlas-test-shard-00-00-vll9l.mongodb.net:38128,' +
           'a-compass-atlas-test-shard-00-01-vll9l.mongodb.net:38128,' +
-          'a-compass-atlas-test-shard-00-02-vll9l.mongodb.net:38128/admin?' +
+          'a-compass-atlas-test-shard-00-02-vll9l.mongodb.net:38128/<DATABASE>?' +
```

Comment:
https://github.com/mongodb-js/connection-model/commit/fa4f56a89478928819e612b1f04a8845df73d104 and https://en.wikipedia.org/wiki/Zero_one_infinity_rule